README.md

```diff
@@ -1,3 +1,44 @@
-# crypto
+<!-- logo -->
+<p align="center">
+  <img src="logo.png">
+</p>
 
-Crypto used in the Sorare stack
+<!-- tag line -->
+<h4 align='center'>JavaScript Crypto helpers for Sorare</h4>
+
+<!-- primary badges -->
+<p align="center">
+  <a href="https://www.typescriptlang.org/">
+    <img src='https://badges.aleen42.com/src/typescript.svg' />
+  </a> 
+  <a href="https://www.npmjs.com/package/@sorare/crypto">
+    <img src='https://img.shields.io/github/package-json/v/@sorare/crypto?label=npm' />
+  </a>
+  <a href="https://github.com/sorare/crypto/actions/workflows/node.js.yml">
+    <img src='https://github.com/sorare/crypto/actions/workflows/node.js.yml/badge.svg' />
+  </a>
+  <a href="https://starkware.co/">
+    <img src="https://img.shields.io/badge/powered_by-StarkWare-navy">
+  </a>
+</p>
+
+`@sorare/crypto` is a JavaScript library providing various crypto functions to be used to sign your Sorare LimitOrder objects in order to make a bid, create or accept an offer. It can be used in both NodeJS and Browser environments.
```

Comment:
```suggestion
`@sorare/crypto` is a Typescript library providing various crypto functions to be used to sign your Sorare LimitOrder objects in order to make a bid, create or accept an offer. It can be used in both NodeJS and Browser environments.
```

I don't know the rule here...

Reply:
Well, I think I'll write "JavaScript library (TypeScript types included)"